src/components/Message.js

```diff
@@ -1,9 +1,7 @@
-function Message (props){
-    return (
-      <>
-        <p className="message">{props}</p>
-      </>
-    );
+function Message(props) {
+  return (
+      <p className="message">{props}</p>
+  );
 }
```

Comment:
if you were to console.log `props` before return, you will see that props is an object. And we cannot directly print object, we need to specify which property of that object we want to print. So` {props.message}` would do the trick, as this is the name we gave in parent component:

`<Message message={props.tweet.message} />`